pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/RawFileKeyReader.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.pulsar.functions.api.examples;
+
+import lombok.Data;
+import org.apache.pulsar.client.api.CryptoKeyReader;
+import org.apache.pulsar.client.api.EncryptionKeyInfo;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+@Data
```

Comment:
Maybe we need to add comments to the newly created class, otherwise the check of ci may fail.